CONTRIBUTING.md

```diff
@@ -17,13 +41,82 @@ version 14 on ubuntu:
 sudo apt-get install postgres-server-dev-14
 ```
 
+## Platform-Specific Installation Instructions
+
+### Windows Installation
+
+If you're working on native Windows, follow these steps to install and build pgmq:
+
+1. Install PostgreSQL for Windows
+
+- Download the official PostgreSQL installer from [PostgreSQL Windows Downloads](https://www.postgresql.org/download/windows/).
+
+- Follow the installation process, ensuring you include pgAdmin and Command Line Tools
+
+- After installation, add the PostgreSQL binary directory (typically `C:\Program Files\PostgreSQL\<version>\bin`) to your system's `PATH` environment variable. This will allow access to PostgreSQL tools like `pg_config`.
+
+2. Install Build Tools (MinGW)
+
+- Download and install [MinGW]() from MinGW. Select the GCC compiler for C/C++
```

Comment:
@Saturn225 it looks like MinGW is missing the url.

Reply:
Added!